src/flag_gems/ops/mm.py

```diff
@@ -146,3 +146,42 @@ def mm(a, b):
             GROUP_M=8,
         )
     return c
+
+
+def mm_out(a, b, *, out):
+    logger.debug("GEMS MM_OUT")
+    # handle non-contiguous inputs if necessary
+    if a.stride(0) > 1 and a.stride(1) > 1:
+        a = a.contiguous()
+    if b.stride(0) > 1 and b.stride(1) > 1:
+        b = b.contiguous()
```

Comment:
It's slightly faster to trim down method calls and operators 

```suggestion
    if 1 not in a.stride():
        a = a.contiguous()
    if 1 not in b.stride():
        b = b.contiguous()
```